tests/conftest.py

```diff
@@ -226,6 +227,57 @@ def fixture(session):
 )
 
 
+def make_spectrum_source_fixture(info):
+    @pytest.fixture
+    def fixture(session):
+        sid = info["source_id"]
+        try:
+            session.create_spectrum_source(
+                sid, meta=info["meta"], channels=info["channels"]
+            )
+            for m in info["measurements"]:
+                session.create_spectrum_measurement(
+                    sid, m["freq_range"], m["timestamp"], m["data"], m["meta"]
+                )
+            yield info
+        finally:
+            session.delete_source(sid)
+
+    return fixture
+
+
+empty_spectrum_source = make_spectrum_source_fixture(
+    {
+        "source_id": uuid.uuid1().hex,
+        "meta": {"meta_test": "empty"},
+        "channels": ["ch"],
+        "measurements": [],
+    }
+)
+
+simple_spectrum_source = make_spectrum_source_fixture(
+    {
+        "source_id": uuid.uuid1().hex,
+        "meta": {"sid_meta": "sid_test"},
+        "channels": ["ch_0", "ch_1"],
+        "measurements": [
+            {
+                "timestamp": 0,
+                "data": {"ch_0": [0.1, 0.2, 0.3], "ch_1": [1, 2, 3]},
+                "freq_range": [2, 10],
+                "meta": {"meta0": "test_0"},
+            },
+            {
+                "timestamp": 100500,
+                "data": {"ch_0": [1.1, 2.2], "ch_1": [1, 2]},
+                "freq_range": [1, 100],
+                "meta": {"meta1": "test100500"},
+            },
+        ],
+    }
+)
+
```

Comment:
It will look more nicer to have meaningful values for the channel, freq_range, and meta.

Reply:
Do you have any suggestions? I guess it would make more sense if we had some strict data validation.

Reply:
It would be nice to generate simulated spectrum measurements. The measurements may not be used actively now, but the future tests might. Resolved this in https://github.com/vikinganalytics/mvg/pull/207/commits/92b884889dcd434a6234a5264b9a68915c356956